recipes-multimedia/libcamera-apps/rpi-libcamera-apps_git.bb

```diff
@@ -0,0 +1,41 @@
+SUMMARY = "A suite of libcamera-based apps for the Raspberry Pi"
+DESCRIPTION = "This is a small suite of libcamera-based apps that aim to \
+copy the functionality of the existing \"raspicam\" apps."
+HOMEPAGE = "https://github.com/raspberrypi/libcamera-apps"
+SECTION = "console/utils"
+
+SRC_URI = "\
+    git://github.com/raspberrypi/libcamera-apps.git;protocol=https;branch=main \
+"
+SRCREV = "22a52590c33a813743b4e6337478c208201c77b1"
+
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://license.txt;md5=a0013d1b383d72ba4bdc5b750e7d1d77"
+
+S = "${WORKDIR}/git"
+
+COMPATIBLE_MACHINE = "^rpi$"
+
+DEPENDS = "libcamera libexif jpeg tiff libpng boost"
+RDEPENDS:${PN} = "python3-core libcamera libexif jpeg tiff libpng"
+
+inherit cmake pkgconfig
+
+EXTRA_OECMAKE = "\
+    -DCMAKE_BUILD_TYPE=Release \
+    -DBoost_INCLUDE_DIR=${STAGING_INCDIR} \
+    -DCMAKE_LIBRARY_PATH=${STAGING_LIBDIR} \
+    ${@bb.utils.contains("DISTRO_FEATURES", "x11", "-DENABLE_X11=1", "-DENABLE_X11=0", d)} \
+    ${@bb.utils.contains("MACHINE", "raspberrypi3", "-DENABLE_COMPILE_FLAGS_FOR_TARGET=armv8-neon", "", d)} \
+    ${@bb.utils.contains("MACHINE", "raspberrypi4", "-DENABLE_COMPILE_FLAGS_FOR_TARGET=armv8-neon", "", d)} \
+"
+
+PACKAGECONFIG[qt] = "-DENABLE_QT=1"
+PACKAGECONFIG[opencv] = "-DENABLE_OPENCV=1"
+PACKAGECONFIG[tensorflow-lite] = "-DENABLE_TFLITE=1"
+
+FILES:${PN} = "${bindir} \
+    ${libdir} \
+"
+SOLIBS = ".so"
```

Comment:
I would add a comment to explain why this is needed. Probably unversioned libraries.

Reply:
Will try without it to see if it works. In the past, I had to manually create symlinks in do_install:append task.

Reply:
Get build error without

```bash
SOLIBS = ".so"
FILES_SOLIBSDEV = ""
```


```log
ERROR: rpi-libcamera-apps-git-r0 do_package_qa: QA Issue: -dev package rpi-libcamera-apps-dev contains non-symlink .so '/usr/lib/libimages.so'
-dev package rpi-libcamera-apps-dev contains non-symlink .so '/usr/lib/libcamera_app.so'
-dev package rpi-libcamera-apps-dev contains non-symlink .so '/usr/lib/libpreview.so'
-dev package rpi-libcamera-apps-dev contains non-symlink .so '/usr/lib/liboutputs.so'
-dev package rpi-libcamera-apps-dev contains non-symlink .so '/usr/lib/libencoders.so'
-dev package rpi-libcamera-apps-dev contains non-symlink .so '/usr/lib/libpost_processing_stages.so' [dev-elf]
```

I'll leave it in and add a comment that it is needed due to unversioned libraries.

Reply:
Indeed. This is becasue the libs are not versioned I was as suspecting. Looks good.